api/proto/config/storage.proto

```diff
@@ -107,6 +109,7 @@ message Drive {
   DriveType drvtype = 8;
   Target target = 9;
   // Initial image need to be resized to this size.
+  // maxsizebytes indicates the maximum size of the volume.
   // A value of 0 will indicate that no resizing is required
```

Comment:
That doesn't read well. Reorder and make the resizing part clearer.

Reply:
Updated.